packages/belos/src/BelosGmresPolySolMgr.hpp

```diff
@@ -298,70 +275,45 @@ class GmresPolySolMgr : public SolverManager<ScalarType,MV,OP> {
 
 private:
 
-  // Method for checking current status test against defined linear problem.
-  bool checkStatusTest();
-
-  // Method for generating GMRES polynomial.
-  bool generatePoly();
-
   // Linear problem.
   Teuchos::RCP<LinearProblem<ScalarType,MV,OP> > problem_;
 
   // Output manager.
-  Teuchos::RCP<OutputManager<ScalarType> > printer_;
   Teuchos::RCP<std::ostream> outputStream_;
 
-  // Status test.
-  Teuchos::RCP<StatusTest<ScalarType,MV,OP> > sTest_;
-  Teuchos::RCP<StatusTestMaxIters<ScalarType,MV,OP> > maxIterTest_;
-  Teuchos::RCP<StatusTest<ScalarType,MV,OP> > convTest_;
-  Teuchos::RCP<StatusTestResNorm<ScalarType,MV,OP> > expConvTest_, impConvTest_;
-  Teuchos::RCP<StatusTestOutput<ScalarType,MV,OP> > outputTest_;
-
-  // Orthogonalization manager.
-  Teuchos::RCP<MatOrthoManager<ScalarType,MV,OP> > ortho_;
-
   // Current parameter list.
   Teuchos::RCP<Teuchos::ParameterList> params_;
+  Teuchos::RCP<Teuchos::ParameterList> outerParams_;
 
   // Default solver values.
   static constexpr int maxDegree_default_ = 25;
-  static constexpr int maxRestarts_default_ = 20;
-  static constexpr int maxIters_default_ = 1000;
-  static constexpr bool strictConvTol_default_ = false;
-  static constexpr bool showMaxResNormOnly_default_ = false;
-  static constexpr int blockSize_default_ = 1;
-  static constexpr int numBlocks_default_ = 300;
   static constexpr int verbosity_default_ = Belos::Errors;
-  static constexpr int outputStyle_default_ = Belos::General;
-  static constexpr int outputFreq_default_ = -1;
-  static constexpr const char * impResScale_default_ = "Norm of RHS";
-  static constexpr const char * expResScale_default_ = "Norm of RHS";
   static constexpr const char * label_default_ = "Belos";
+  static constexpr const char * outerSolverType_default_ = "";
+  static constexpr const char * polyType_default_ = "Arnoldi";
   static constexpr const char * orthoType_default_ = "DGKS";
```

Comment:
See above comment on the default orthogonalizer type.

Reply:
See above.